config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :permits
```

Comment:
Should be in the API namespace and limited to the routes actually used (index and show).